packages/roosterjs-plugin-image-resize/lib/ImageResize.ts

```diff
@@ -223,7 +229,7 @@ export default class ImageResize implements EditorPlugin {
         parent.insertBefore(document.createComment(END_TAG), resizeDiv.nextSibling);
 
         resizeDiv.style.position = 'relative';
-        resizeDiv.style.display = 'inline-table';
+        resizeDiv.style.display = 'inline-flex';
```

Comment:
Please make sure this also works for IE11.

Reply:
yes, this is working on IE11 for me